background/logging.js

```diff
@@ -0,0 +1,32 @@
+require('app-module-path').addPath(__dirname + '../../app');
+
+var path = require('path');
+
+var nconf = require('nconf');
+var bunyan = require('bunyan');
+
+formatBunyanInfoLog = function() {};
+formatBunyanInfoLog.prototype.write = function (rec) {
+    console.log('[%s] %s: %s',
+        rec.time,
+        bunyan.nameFromLevel[rec.level],
+        rec.msg);
+};
+
+logger = bunyan.createLogger({
+    name: 'Bitcannon',
```

Comment:
How should logging be handled?

We could allow the log level and format strings to be customised in the config.json and perhaps allow external loggers via a logging class / module of some sort that can be swapped out for others as long as they conform to a common interface.

>This should also be showing that the logs are coming from the web interface since we wanted to modalise this and break it into separate processes.

Do you mean show logs from the web interface (server) and imports at the same time? How could we go about doing this?